README.MD

```diff
@@ -1,42 +1,258 @@
-# This is the official TeaSpeak docker reposetory
-## Build the debian docker
-### Debian
-Just build the docker by executing:
-```
-docker build --tag teaspeak:debian server/debian/
-```
-Running the docker:
-```
-docker run --rm -i teaspeak:debian
-```
-Run with parameters:
+# This is the official TeaSpeak docker repository
+This repository holds all Dockerfiles and bash scripts to build the TeaSpeak Server Images as well as the Web-Client Image
+
+## Table of Content
+1. [Server Images](#server-images)
+    1. [Build and Run - Alpine Flavoured](#alpine-179mb-16-layers)
+    2. [Build and Run - Debian Flavoured](#debian-427mb-14-layers)
+    3. [Build and Run - Ubuntu Flavoured](#ubuntu-431mb-17-layers)
+    6. [EnvVars and Build-Args](#envvars-and-build-args-server)
+    4. [Run the Server via `docker-compose`](#run-the-server-via-docker-compose)
+    5. [View the Logs and get the Token](#view-the-logs-and-token)
+2. [Web Image](#web-image)
+    1. [Build and Run - Alpine Flavoured](#alpine-413mb-21-layers)
+    2. [Files and Volumes](#files-and-volumes)
+    3. [EnvVars and Build-Args](#envvars-and-build-args-web)
+    4. [Host the WebClient via `docker-compose`](#host-the-web-client-via-docker-compose)
+    5. [Ideas for the Future](#ideas-to-improve-the-web-client-image)
```

Comment:
`Ideas for the Future`, they're actiually `Ideas for future improvements [of the WebClient image]`.

Reply:
I restructured that part, it should be clearer now